pom.xml

```diff
@@ -33,7 +33,7 @@
 	</modules>
 	
 	<properties>
-		<openMRSVersion>1.12.0-SNAPSHOT</openMRSVersion>
+		<openMRSVersion>1.11.3</openMRSVersion>

```

Comment:
I think that we need to keep the 1.12 snapshot as some of the optimisations are only in those version of OpenMRS.